### PR TITLE
Adds force-start operation to cluster.sh script.

### PR DESCRIPTION
**Pull request by kobalski**
Sample usage : 
`sh cluster.sh --operation force-start`

### CODE REVIEW

**Comment by kobalski** [comment]
run-lab-run

**Comment by mdogan** [comment]
:+1: 

no need to run or wait for full build, this is only a script change.

File: hazelcast/src/main/resources/cluster.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 if [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
    	echo "parameters : "
-   	echo "	-o, --operation	    : Defines state operation of the script. Operation can be 'get' or 'change'."
+   	echo "	-o, --operation	    : Defines state operation of the script. Operation can be 'get','change','shutdown','force-start'."
```
Comment (mesutcelik) [inline hazelcast/src/main/resources/cluster.sh]: `get-state`,`change-state` would be better names for those operations.

Another naming suggestion :
`Defines state operation of the script` --> `executes cluster-wide operation`

File: hazelcast/src/main/resources/cluster.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
     echo "	-s, --state 	    : Defines new state of the cluster. State can be 'active', 'frozen', 'passive'"
```
Comment (mesutcelik) [inline hazelcast/src/main/resources/cluster.sh]: `Defines new state of the cluster` --> `updates cluster state` or something similar.

File: hazelcast/src/main/resources/cluster.sh (hunk 1 above)
Comment (mdogan) [inline hazelcast/src/main/resources/cluster.sh]: :+1:

File: hazelcast/src/main/resources/cluster.sh [hunk 3]
```diff
@@ -131,6 +131,33 @@ if [ "$OPERATION" = "change" ]; then
     exit 0
 fi
 
+if [ "$OPERATION" = "force-start" ]; then
+    echo "You are force starting the cluster. Please make sure you provide valid user/pass."
```
Comment (mesutcelik) [inline hazelcast/src/main/resources/cluster.sh]: `You are force starting the cluster`. Any better explanation for this @mdogan @metanet ?

Comment (mesutcelik) [inline hazelcast/src/main/resources/cluster.sh]: `force-start makes cluster operational if cluster start is blocked by problematic members.` could be an option

**Comment by kobalski** [comment]
Merging this, PR will update force update info message with direct commit.

**Pull request merged**